base/deprecated.jl

```diff
@@ -209,7 +209,7 @@ for deprecatedfunc in [:isprime, :primes, :primesmask, :factor]
     @eval begin
         $deprecatedfunc(args...) = error(string($deprecatedfunc, args,
             " has been moved to the package Primes.jl.\n",
-            "Run Pkg.add(\"Primes\") to install Primes on Julia v0.5-"))
+            "Run Pkg.add(\"Primes\") to install Primes on Julia v0.5- and then run `using Primes`."))
```

Comment:
> FWIW that's not what 0.5- means but since 0.5 is released this can be changed now.

Sorry -- what then does `0.5-` mean and how should the migration messages using that terminology read? Thank you!